### DISCUSSION TITLE
Strange data with no gps fix and good locator and height

### DISCUSSION THREAD

**Discussion by whallmann** [EngineerGuy314/pico-WSPRer#34] [General]
I started today this wspr ballon. channel 478 DK7PE (a friends call).
After some time the gps fix shows null but height and encoded 5+6 locator grids are well.
find it at 2024-08-05  09.14 UTC
Same config was in air bevor abt 16 days.  Cant explain this behaivor.
73 Wolfgang DF7PN 
[page of dk7pe](https://traquito.github.io/search/spots/dashboard/?band=20m&channel=478&callsign=DK7PE&dtGte=2024-08-04&dtLte)

**Comment by EngineerGuy314** [comment]
I think its possible that it had a very marginal GPS signal.  It transmit a '0' for GPS valid if there is no position fix at the moment that the 1st packet is being generated. But if the GPS fix even momentarily goes on again during the next 10 minutes then the altitude and grid56 values will get updated will new values.

What kind of GPS antenna did you use? (the value of knots, divided by 2, is the number of satellites that are seen).  So at 7:24utc, knots were 10, meaning only 5 satellites were being used for the fix, which is very low. It seems likely you were losing and  re-establishing GPS fix rapidly.

if you enable TELEN type-5, it will report minutes-since-GPS-fix-was-obtained.  I haven't been able to flight test that yet, but my intention was for that to give another data-point to diagnose problems from GPS loss.  Specifically, I lose a lot of spots over Russia, and I'm never sure if its from GPS loss due to jamming, or just sparsity of reporting stations in that region.

**Reply by whallmann** [reply]
Hm,  thanks.  On the floor there were most the time over 15 Sats (max 19). So it was at start, also on last start with same GPS unit last week. But in one moment 11 Sats then 0  - i cant handle with this big changes.
The antenna is a simple 9 cm wire fixed with hot glue after soldering.  I think more and more that the HF is disturbing something this time. But its only a thought.  I am asking myself what can i do to make it better on next flight.   So the floor check was excellent.

Picture:  blue cable is top antenna, yellow bottom - black platic hut is open to all sides - just sun shield to avoid overheated cpu

![IMG_20240804_151203](https://github.com/user-attachments/assets/0fbe11f3-410b-45cb-bbe2-e2c4d750381c)

**Reply by EngineerGuy314** [reply]
if you are using just a monopole for the gps antenna, I recommend 46mm.  It has worked very well for me.

**Comment by whallmann** [comment]
Sounds good. Will have this on the next try.

EngineerGuy314 ***@***.***> schrieb am Di., 6. Aug. 2024,
03:45:
